Lombiq.BaseTheme/Views/Layout.cshtml

```diff
@@ -22,32 +22,32 @@
     page: this,
     new[]
     {
-        new ZoneDescriptor(Header, wrapBody: true, elementName: "header")
+        new ZoneDescriptor(Header, wrapBody: true, elementName: "header", ariaLabel: "Site")
```

Comment:
`aria-label` is only for accessibility right? If so, then these should be localized. I suggest changing the property type of `ZoneDescriptor.AriaLabel` from `string` to `LocalizedHtmlString`.

Reply:
Yep, and good idea!